### ISSUE TITLE
Weights not used in training batch

### ISSUE THREAD

**Issue by hugokitano** [thtang/CheXNet-with-localization#14]
Hi! Thanks for this project. I noticed the weights for each training batch are not used. How did you incorporate them in training? I read in your report that your mean AUC ended up being better with them.

**Comment by thtang** [comment]
What do you mean **_weights for each training batch_** ?

**Comment by hugokitano** [comment]
![image](https://user-images.githubusercontent.com/8139124/75913262-73df0880-5e07-11ea-8e97-a98e3fc8d0ca.png)

In train.py, you define weights_sub, but they are not used for anything in the loss function.

**Comment by thtang** [comment]
Yup, the weights are originally used for loss function as described in the paper. However, **there is no improvement regarding AUC score** but makes the training code messy. If you've read different versions of the paper on arXiv, you will find out that they obtained the definitely same AUC number using different loss functions.
